frontend/src/components/layout/Layout.module.scss

```diff
@@ -167,7 +167,11 @@ $toastify-toast-close-button: var(--toast-close-button);
       display: inline-flex;
       align-items: center;
       gap: $spacing-sm;
-      padding: $spacing-md $spacing-xl;
+      padding: $spacing-md;
+
+      @media screen and (min-width: $content-xl) {
```

Comment:
This one I will leave up to @maxxcrawford - but the reason I used that here was because it aligns the logo with the content @ 1152px - snap it in place essentially (see video). Or we can modify line 171 to use the same padding and then deal with the other side as well.